packages/astro/components/Code.astro

```diff
@@ -94,60 +93,13 @@ if (typeof lang === 'object') {
 
 const highlighter = await getCachedHighlighter({
 	langs: [lang],
-	themes: Object.values(experimentalThemes).length ? Object.values(experimentalThemes) : [theme],
+	theme,
+	experimentalThemes,
+	wrap,
 });
```

Comment:
With the abstracted function, we can pass the individual options directly, and it'll handle and process them internally.